Ui/View/Host/ProtocolHosts/AxMsRdpClient09Host.xaml.cs

```diff
@@ -322,11 +322,6 @@ private void RdpInitRedirect()
                 // - 1 (Audio redirection is enabled and the option is "Leave at remote computer". The "Leave at remote computer" option is supported only when connecting remotely to a host computer that is running Windows Vista. If the connection is to a host computer that is running Windows Server 2008, the option "Leave at remote computer" is changed to "Do not play".)
                 // - 2 (Audio redirection is enabled and the mode is "Do not play".)
                 _rdpClient.SecuredSettings3.AudioRedirectionMode = 0;
```

Comment:
we will set `_rdpClient.AdvancedSettings8.AudioQualityMode` only when `AudioRedirectionMode` is `RedirectToLocal`